assets/src/dashboard/components/scrollToTop/index.js

```diff
@@ -88,14 +72,15 @@ const ScrollToTop = () => {
   }, [handleScroll]);
 
   return (
-    <ScrollButton
+    <StyledButton
```

Comment:
do we need any `aria-hidden` labels on here when it's not visible?

Reply:
Yeah, what happens when the button is not visible? Could we perhaps just not render it? That's a lot better for a11y (and semantics) - so just `{isVisible && <StyledButton />}`.

Reply:
That would add technical complexity for animating in & out. But idk how important those animations are if we're fine with getting rid of em 🤷

Reply:
^^ This is a good question to run past Sam. If she wants the animation to stay, there's our answer.

Reply:
Ehh I wouldn't let this hold this PR up if it's been up for a while. I think either of the proposed comments would be more of a a11y enhancement and the purpose of this PR is not to clean up the implementation, but to just bring in components from the ds-system

Reply:
v true, max - thanks for staying in scope :)

Reply:
For now I added `aria-hidden={isVisible}` since `isVisible` is the state we're using to determine when to animate the button in.

If we want to get rid of the animation I can do that as well.